mysql-test/suite/rpl/t/bug85158.test

```diff
@@ -0,0 +1,109 @@
+#
+# Testcase for bug 85158.
+#
+--source include/not_group_replication_plugin.inc
+--source include/have_gtid.inc
+--source include/have_perfschema.inc
+--source include/have_binlog_format_row.inc
```

Comment:
Old question still open:

laurynas-biveinis on Sep 22  Member
Does the bug reproduce under row only or is it simply binlog format-independent? In the latter case let's just run it under all three formats unless its runtime is very long
      @vlad-lesin
vlad-lesin on Sep 26  Member
yep
      @laurynas-biveinis
laurynas-biveinis on Oct 5  Member
"yep" as in "row only"? The testcase is still limited to that format

Reply:
Yes, I missed it. There is no need to limit the test with row-based binlog format case.